site/content/en/docs/Guides/access-api.md

```diff
@@ -134,6 +134,131 @@ kubectl logs create-gs-6wz86-7qsm5  --namespace agones-system
 ```
 You have just created a GameServer using Kubernetes Go Client.
 
+## What are Informers and Listers? How can I use these?
+
+Informers and Listers are mechanism for Kubernetes.
+Almost all of the native Kubernetes controllers and custom controllers use these mechanisms.
+Kubernetes's control plane can be accessed via Kubernetes APIs, however,
+the load for control plane cannot be ignored when the client program accesses to the control plane every time.
+
+Informers and Listers can reduce the load for the control plane by using the in-memory-cache.
+When Informes and Listers are created, the in-memory-cache called as the store are also created.
+The client program accesses and queries to the in-memory-cache, not control plane,
+which enables reducing the load.
```

Comment:
```suggestion
Informers and Listers reduce the load on the Kubernetes control plane by creating, using and maintaining and eventually consistent an in-memory-cache. This can be watched and also queried with zero cost, since it will only read against its in-memory model of the Kubernetes resources.
```